conf/testOnlyDoNotUseInAppConf.routes

```diff
@@ -42,3 +42,9 @@ GET           /submissions/test-only/:formTemplateId/:destinationId/:accessCode
 GET           /submissions/gform-plugin/exprs/:formTemplateId             uk.gov.hmrc.gform.testonly.DebugController.exprs(formTemplateId: FormTemplateId)
 GET           /submissions/gform-plugin/model/:formTemplateId             uk.gov.hmrc.gform.testonly.DebugController.model(formTemplateId: FormTemplateId)
 GET           /submissions/gform-plugin/verify/:formTemplateId            uk.gov.hmrc.gform.testonly.DebugController.verify(formTemplateId: FormTemplateId)
+
+GET           /submissions/errors/:formTemplateId  uk.gov.hmrc.gform.testonly.TestOnlyErrorMessageController.errorMessages(formTemplateId: FormTemplateId, a: Option[AccessCode], fullReport: Boolean, jsonReport: Boolean  )
+GET           /submissions/errors-json-full/:formTemplateId  uk.gov.hmrc.gform.testonly.TestOnlyErrorMessageController.errorsJsonFull(formTemplateId: FormTemplateId, a: Option[AccessCode])
+GET           /submissions/errors-json/:formTemplateId  uk.gov.hmrc.gform.testonly.TestOnlyErrorMessageController.errorsJson(formTemplateId: FormTemplateId, a: Option[AccessCode])
+GET           /submissions/errors-html-full/:formTemplateId  uk.gov.hmrc.gform.testonly.TestOnlyErrorMessageController.errorsHtmlFull(formTemplateId: FormTemplateId, a: Option[AccessCode])
```

Comment:
I tried to implement the ability to produce not only report of the potential error messages for the particular form component from the template but also for all potential errors for the components in general. (somehting like to display all errors when  for example mandatory is on or off). For that I have slightly complicated syntax in our DSL (`cond` and `andCond` in ifProgram and switchCases. It works but now I think it might be a bit difficult to follow and may be it is better to remove it.

Reply:
I think report should be just one and it should exactly match state of the component. Ie. if something is mandatory off, report should not give you a glimpse of how it would look like if it was mandatory on.